test/jdk/java/net/httpclient/HttpResponseLimitingTest.java

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
```

Comment:
Hello Volkan, for the test files, we don't use the "Classpath" exception copyright header. You can copy over the copyright text from another existing test file, to get the correct header.

Reply:
Fixed in ad9de094675f869d5df28530cf41f17acdda0bd5.